src/static/css/custom_common.css

```diff
@@ -664,8 +664,14 @@ to overlap borders ****/
     border-radius: 0px;
 }
 
+/*
+The .list-group-item-buttons rule sets a minimum height of 65px.
+This is necessary to eliminate a small gap that appears between an expanded
+bootstrap-table accordion row and the subsequent row. This gap is caused by a
+15px margin-bottom on the .pull-right child element of .list-group-item-buttons.
+*/
 .list-group-item-buttons {
-    min-height: 55px;
+    min-height: 65px;
```

Comment:
As I mentioned in my comment:

> I think the better solution is to set margin-bottom: 10 px; on the buttons, via classes shown here:
> 
![image](https://github.com/bytedeck/bytedeck/assets/10604391/80d4f752-2a65-4d50-b1d8-14edce124b0a)


Setting the min height to 65, although it solves the gap problem just makes everything unnecessarily bigger than it needs to be and leaves a big gap below if there is only a small amount of text.